### PR TITLE
Custom object matcher in use

### PR DESCRIPTION
**Pull request by renra**

### CODE REVIEW

**Comment by renra** [comment]
**RDY**

**Comment by juanramoncg** [comment]
👍 **LGTM** Nice to see this new matcher

File: spec/segment_spec.js [hunk 1]
```diff
@@ -11,10 +12,14 @@ function matchSegmentCall(actualData, expectedData) {
     body = JSON.parse(actualData.body);
 
   expect(method).toEqual(expectedData.method);
-  expect(headers).toEqual(expectedData.headers);
-  expect(body).toEqual(expectedData.body);
+  expect(headers).toEqualObject(expectedData.headers);
+  expect(body).toEqualObject(expectedData.body);
 }
 
+beforeEach(function(){
```
Comment (juanramoncg) [inline spec/segment_spec.js]: `Nit` maybe just `beforeAll` will work?

Comment (renra) [inline spec/segment_spec.js]: 👍 It does, seems better indeed. I'm still missing some master jasmine file - something like `spec_helper` in `rspec` where we could put these things globally.

Comment (juanramoncg) [inline spec/segment_spec.js]: Definitely, i think we should able to do something like that and link the file in here: https://github.com/GlobalWebIndex/susi/blob/master/jest.conf

**Pull request merged**